pgql-lang/src/main/java/oracle/pgql/lang/SpoofaxAstToGraphQuery.java

```diff
@@ -204,10 +202,6 @@ public static PgqlStatement translate(IStrategoTerm ast) throws PgqlException {
         return translateCreatePropertyGraph(ast);
       case "DropPropertyGraph":
```

Comment:
Isn't it necessary to add create super property graph to this case?

Reply:
at this point it is still part of "CreatePropertyGraph", sine I extended the existing "CreatePropertyGraph" grammar rule here: https://github.com/oracle/pgql-lang/pull/114/files/e7262af3e90f98aab3d9ab489aecc93a07193aa7#diff-bbe8c5457243c81eb25882184b2f2f279e08c53fcf43ef554557d8425efc1cd7R15

Only later we split things up into two separate types of statements. Reason I created a new type of statement in the IR is so that we will automatically get an error message in PGX / PG_SCHEMA / PG_SQL for it rather than the base graphs being silently ignored if we forget to check for them.